Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM ubuntu:16.04
-# If host is running squid-deb-proxy on port 8000, populate /etc/apt/apt.conf.d/30proxy
-# By default, squid-deb-proxy 403s unknown sources, so apt shouldn't proxy ppa.launchpad.net
-RUN awk '/^[a-z]+[0-9]+\t00000000/ { printf("%d.%d.%d.%d\n", "0x" substr($3, 7, 2), "0x" substr($3, 5, 2), "0x" substr($3, 3, 2), "0x" substr($3, 1, 2)) }' < /proc/net/route > /tmp/host_ip.txt
-RUN perl -pe 'use IO::Socket::INET; chomp; $socket = new IO::Socket::INET(PeerHost=>$_,PeerPort=>"8000"); print $socket "HEAD /\n\n"; my $data; $socket->recv($data,1024); exit($data !~ /squid-deb-proxy/)' <  /tmp/host_ip.txt \
-  && (echo "Acquire::http::Proxy \"http://$(cat /tmp/host_ip.txt):8000\";" > /etc/apt/apt.conf.d/30proxy) \
-  && (echo "Acquire::http::Proxy::ppa.launchpad.net DIRECT;" >> /etc/apt/apt.conf.d/30proxy) \
-  || echo "No squid-deb-proxy detected on docker host"
-RUN dpkg --add-architecture i386 \
-  && apt-get update \
-  && apt-get install -y software-properties-common \
-  && add-apt-repository ppa:pipelight/stable \
-  && apt-get update \
-  && apt-get install -y --install-recommends wine-staging cabextract wget python-dev build-essential git
-RUN wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py
-RUN pip install --upgrade requests
+FROM python:slim
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends wget git \
+    && rm -rf /var/lib/apt/lists/*
+
 RUN useradd -d /home/user -s /bin/bash -m user
 WORKDIR /home/user
 RUN mkdir dump_syms
-ADD dump-syms.manifest /home/user/dump_syms/
-ADD config.py.docker /home/user/config.py
-ADD https://hg.mozilla.org/mozilla-central/raw-file/default/python/mozbuild/mozbuild/action/tooltool.py /home/user/tooltool.py
+RUN wget https://bootstrap.pypa.io/get-pip.py \
+    && python get-pip.py \
+    && wget -qO- https://github.com/mozilla/dump_syms/releases/latest/download/dump_syms-linux-x86_64.tar.gz | tar xvz -C dump_syms
+
+ADD requirements.txt /home/user/
+RUN pip install --upgrade -r requirements.txt
+
+RUN apt-get remove -y wget && rm get-pip.py && python -m pip uninstall pip -y
```

Comment:
This is not going to help, as this Docker layer will be the same as before + the removal, so in the end the used space wil be the same (see https://forums.docker.com/t/why-run-command-which-deletes-files-inflates-image-size/33670).
So the only way to achieve this is to have a big command where you do everything with &&

Reply:
A bit like https://github.com/mozilla/bugbug/blob/master/infra/dockerfile.base#L5-L10, or https://github.com/mozilla/code-review/blob/master/bot/docker/Dockerfile#L6 to avoid the && but achieve the same result